src/devices/Vcnl4040/Definitions/PowerState.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Iot.Device.Vcnl4040.Definitions
+{
+    /// <summary>
+    /// Defines the set of ALS and SD power states.
+    /// Documentation: datasheet (Rev. 1.7, 04-Nov-2020 9 Document Number: 84274).
+    /// </summary>
+    public enum PowerState : byte
```

Comment:
why do you want to make this one public? You've been implementing a function to hide it with PowerOn?

Reply:
You're right. That's a mistake. This will be internal.